openapi.go

```diff
@@ -454,6 +453,10 @@ func parseStructTags(t reflect.Type, schemaRef *openapi3.SchemaRef) {
 			slog.Warn("Property not found in schema", "property", jsonFieldName)
 			continue
 		}
+		if field.Type.Kind() == reflect.Struct {
+			fieldType := field.Type
+			parseStructTags(fieldType, property)
```

Comment:
You added no tests about validating the error returned by parseStruct that are logged.

You should check they are reported

Reply:
Add a test function using slogassert for warn cases